Formula/g/goat.rb

```diff
@@ -0,0 +1,18 @@
+class Goat < Formula
+  desc "General purpose AT Protocol CLI in Go"
+  homepage "https://github.com/bluesky-social/goat"
+  url "https://github.com/bluesky-social/goat/archive/refs/tags/v0.2.0.tar.gz"
+  sha256 "a0a12d6e767f07039970b61dd855b537404f380c7468a42b1929e5e9f3ff57be"
+  license any_of: ["MIT", "Apache-2.0"]
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = "-s -w -X main.Version=#{version}"
```

Comment:
It seems that the `main.Version` didn't work, it is only work with `.git` folder. Are there any other way to pass the version info correctly?